gdcdictionary/schemas/sample.yaml

```diff
@@ -604,7 +602,6 @@ properties:
       - Peritumoral
       - Unknown
       - Not Reported
-      - Not Allowed To Collect
   tumor_code:
```

Comment:
Also remove `tumor_code` and `tumor_code_id`?

Reply:
Yeah unless you want to repurpose the tumor_code to represent the tumor from which the sample was derived. I went kind of overboard on BloodPAC where I took disease_type and primary_site off case and moved them down here; then I used tumor code as a way to generally characterize the morphology of the tumor.

Reply:
Yeah, I think if we can leave the disease information in diagnosis and then link diagnosis to the samples that's probably the best way forward here. We do likely want to remove `disease_type` and `primary_site` off of case because there's more of a mixture in KF, but can make that a separate issue.

Reply:
Removed `tumor_code` and `tumor_code_id`.